src/metabase/query_processor/postprocess.clj

```diff
@@ -50,38 +30,40 @@
   Where `rff` has the form
 
     (f metadata) -> rf"
-  #_{:clj-kondo/ignore [:deprecated-var]}
-  [(ensure-legacy #'format-rows/format-rows)
-   (ensure-legacy #'results-metadata/record-and-return-metadata!)
-   (ensure-pmbql #'limit/limit-result-rows)
-   (ensure-legacy #'qp.middleware.enterprise/limit-download-result-rows)
-   (ensure-legacy #'qp.add-rows-truncated/add-rows-truncated)
-   (ensure-legacy #'qp.add-timezone-info/add-timezone-info)
-   (ensure-legacy #'qp.middleware.enterprise/merge-sandboxing-metadata)
-   (ensure-legacy #'qp.add-remaps/remap-results)
-   (ensure-legacy #'pivot-export/add-data-for-pivot-export)
-   (ensure-legacy #'large-int/convert-large-int-to-string)
-   (ensure-legacy #'viz-settings/update-viz-settings)
-   (ensure-legacy #'qp.cumulative-aggregations/sum-cumulative-aggregation-columns)
-   (ensure-pmbql #'annotate/add-column-info)
-   (ensure-legacy #'fetch-source-query/add-dataset-info)])
+  [[::legacy #'format-rows/format-rows]
```

Comment:
updated this so we only convert the MBQL 5 query to legacy once and then pass in one version or the other based on the key here... post-processing middleware can't modify the query so this optimization means we don't need to do needless conversion back-and-forth as we start converting more of this middleware to Lib